tools/build/AppImage/.gitignore

```
@@ -0,0 +1,4 @@
+AppDir
+*.txt
+*.AppImage
```

*.AppImage-SHA256.txt
```suggestion
*.AppImage-SHA256.txt
```